CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.17.2...master
 
+### Fixed
+- Correct issue comment link reading to work with Jira versions prior to 8.17.0. Fix [JPERF-814].
```

Comment:
Hmm, but [CI is green for 8.5.0 on master](https://github.com/atlassian/jira-actions/runs/7087593863?check_suite_focus=true).

Reply:
Yup, this indeed is strange. Needs investigation.

Reply:
I'm gonna debug `JiraCoreScenarioIT` and see why the problem doesn't occur.

For more context: The problem started to happen inside `JiraSoftwareScenarioIT` when I bumped jira-actions in jira-software-actions and run it's tests locally.

Reply:
Bad new is that the `JiraCoreScenarioIT` doesn't seem to test anything: https://ecosystem.atlassian.net/browse/JPERF-815

Good news is that we know about it now and we can fix it.

Reply:
I will push a branch proving the issue

Reply:
Pushed here https://github.com/atlassian/jira-actions/pull/109

Reply:
Whoa! :exploding_head: Great catch!

Reply:
Interesting idea. 

I was also considering contributing a `JqlRememberingProjectMemory` which would be built from both `ProjectMemory` and `JqlMemory`, so that when remembering a project it also would remember JQL like this
```
project = ${project.key} order by issuekey desc
``` 

I'm personally using such JQLs frequently. Possibly typical Jira user too?

A code for this memory is very simple
```
package com.atlassian.performance.tools.jiraactions.api.memories.adaptive

import com.atlassian.performance.tools.jiraactions.api.memories.JqlMemory
import com.atlassian.performance.tools.jiraactions.api.memories.Project
import com.atlassian.performance.tools.jiraactions.api.memories.ProjectMemory

class JqlRememberingProjectMemory(
    private val delegate: ProjectMemory,
    private val jqlMemory: JqlMemory
) : ProjectMemory {
    override fun recall() = delegate.recall()

    override fun remember(memories: Collection<Project>) = delegate.remember(memories)
        .also {
            jqlMemory.remember(memories.map { "project = ${it.key} order by issuekey desc" })
        }
}
```

Reply:
Ship it :rocket: